src/containers/modal-segment/index.js

```diff
@@ -0,0 +1,231 @@
+/* --------------------------------------------------------
+* Author Võ Bách Nhạc
```

Comment:
Please set author, your number phone, 
- remove console.log in this file

Reply:
fixed